modules/dr-restoring-cluster-state.adoc

```diff
@@ -304,3 +304,21 @@ etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0
 ----
 
 Note that it might take several minutes after completing this procedure for all services to be restored. For example, authentication by using `oc login` might not immediately work until the OAuth server pods are restarted.
+
+[id="dr-scenario-2-restoring-cluster-state_storage_{context}"]
+= Issues and workarounds in restoring persistent storage state
+
+If your {product-title} uses persistent storage of any form, typically a state of the world is stored outside etcd. It could be a ElasticSearch
+cluster running in a Pod or a database running in a statefulset. When {product-title} is restored from etcd backup, we are also restoring status of those workloads in Openshift. But depending on how old the etcd snapshot is - that status may be invalid or outdated.
+
+Some examples are:
+
+1. Mysql database was running in a Pod backed up by a PV object. But after the etcd snapshot was taken - the Mysql Pod and corresponding PV were deleted and hence underlying volume on storage provider would be deleted. Restoring Openshift from etcd snapshot is not going to bring back the volume on storage provider and hence is not going to result in running Mysql Pod(even though Pod will try to start but will keep failing) and a manual action must be taken to restore mysql Pod. In this case manual action could be restoring the volume on storage provider via volume snapshots and then editing PV object to point to new volume.
```

Comment:
I'd avoid starting the sentence with `But`. This sentence could read - "After the etcd snapshot was taken the Mysql Pod and corresponding PV were deleted, and the underlying volume on the storage provider was also deleted."

Personal nitpick, but I'd reword the rest of this bit to say - "Restoring OpenShift from an etcd snapshot is not going to bring back the volume on the storage provider, and will not result in a running Mysql Pod, despite the Pod repeatedly attempting to start. A manual action must be taken to restore this Pod, such as restoring the volume on the storage provider and then editing the PV to point to the new volume."

Reply:
What are the example of? Stale ectd backups? If so, provide more clarification in the lead-in sentence "Some examples are:"

Reply:
Introduce the term "PV" and lowercase for "pod". So..."Mysql database was running in a pod backed up by a persistent volume (PV) object."

Reply:
I like a lot of Christian's rewrite above, but would recommend a few changes to:
"Restoring OpenShift from an etcd snapshot does not bring back the volume on the storage provider, and does not produce a running Mysql pod, despite the pod repeatedly attempting to start. You must manually restore this pod by restoring the volume on the storage provider, and then editing the PV to point to the new volume."